c2corg_ui/static/js/map/map.js

```diff
@@ -237,6 +237,8 @@ app.MapController = function($scope, mapFeatureCollection, ngeoLocation,
    */
   this.view_ = this.map.getView();
 
+  this.scope_.$root.$on('updateMapResize', this.updateMapResize_.bind(this));
```

Comment:
Please move this line inside the ``if (this.advancedSearch) {`` block, a few lines below (line 250) => we don't need to add this listener whenever a map is created. Only for advanced search pages (perhaps for some other maps later but that still hypothetical => we will make the change later if needed).